examples/step-97/doc/results.dox

```diff
@@ -0,0 +1,116 @@
+| 1 | 6 |  4625 |  111300 | 4.41e-09 | -    |
+| 1 | 7 |  7992 |  192240 | 3.11e-09 | 1.91 |
+| 1 | 8 | 12691 |  305172 | 2.23e-09 | 2.18 |
+| 1 | 9 | 18944 |  455424 | 1.71e-09 | 1.96 |
+| 2 | 6 |  4625 |  375300 | 1.84e-10 | -    |
+| 2 | 7 |  7992 |  648324 | 1.03e-10 | 3.21 |
+| 2 | 8 | 12691 | 1029294 | 6.08e-11 | 3.40 |
+| 2 | 9 | 18944 | 1536192 | 4.04e-11 | 3.07 |
+
+**Table 2. Convergence table. Magnetic field, $\vec{B}$.
+
+The following notations were used in the headers of the tables:
+
+- p - the degree of the finite elements.
+
+- r - the mesh refinement parameter, i.e., the number of nodes on the transfinite
+lines.
+
+- cells - the total amount of active cells.
+
+- dofs - the amount of degrees of freedom.
+
+-$\|e\|_{L^2}$ - the $L^2$ error norm.
+
+-$\alpha_{L^2}$ - the order of convergence of the $L^2$ error norm.
+
+If `Settings::log_cg_convergence = true`, the program saves the convergence data
+of the CG solver into csv files.
+
+The vector representations of the calculated vector fields, $\vec{J}_f$ and
+$\vec{B}$, are illustrated above by the first figure on this page. The figures
+below illustrate slices of the magnitudes of these fields. The figures below
+were simulated with $p = 2$ and $r = 9$. Visual inspection of the vector
+potentials is not very informative as their conservative portions are unknown.
+
+@htmlonly
+<p align="center">
+  <img src="images/steps/developer/step-97-Jf.svg" alt="The result - free-current
+ density" height="531">
+</p>
+@endhtmlonly
+
+@htmlonly
+<p align="center">
+  <img src="images/steps/developer/step-97-B.svg" alt="The result - magnetic field
+ density" height="531">
+</p>
+@endhtmlonly
+
+<h3>Possibilities for extensions</h3>
+
+Repeat the simulations for the three types of the boundary conditions,
+Dirichlet, Neumann, and Robin. The Robin boundary condition is supposed to be
+superior to the other two. Look at the simulated data to see that this is indeed
+the case. You can save the projected exact solution next to the simulated
+solutions into the vtu files, just set `Settings::project_exact_solution = true`.
+ParaView has "Plot Over Line" filter. You can use this filter to visualize the
+difference between the exact solution and a solution simulated with a particular
+boundary condition. You can also draw conclusions by observing the convergence
+tables. Keep in mind the $\eta^2$ parameter. Increase it if the CG solver chocks
+while you are experimenting. Note that the benefits offered by the Robin
+boundary condition are observed the best when higher-order finite elements are
+used, i.e., $p = 1$ and $p = 2$.
+
+The Robin boundary condition as described above is also called the first-order
+asymptotic boundary condition (ABC). There exist ABCs of higher orders. Implement
+and test the second-order ABC to see if it performs any better. There exist
+improvised asymptotic boundary conditions, IABCs. Try to implement the first
+order IABC.
```

Comment:
If you have a reference, this would be a good place to put it to help people find out more about these ABCs.